Remotion/Web/Core/Res/Themes/NovaViso/UI/DatePickerForm.aspx

```diff
@@ -27,15 +27,15 @@
 <body>
   <form id="Form" method="post" runat="server">
   <asp:Calendar ID="Calendar" runat="server" Height="16em" Width="14em" Font-Size="100%" BackColor="White"
-    DayNameFormat="FirstLetter" ForeColor="Black" BorderColor="Black" CellPadding="4">
-    <TodayDayStyle ForeColor="Black" BackColor="#C5E0FC"></TodayDayStyle>
-    <SelectorStyle BackColor="#C5E0FC"></SelectorStyle>
+    DayNameFormat="FirstLetter" ForeColor="Black" BorderColor="#979797" CellPadding="4">
+    <TodayDayStyle ForeColor="Black" BackColor="#DEF3FE"></TodayDayStyle>
+    <SelectorStyle BackColor="#DEF3FE"></SelectorStyle>
     <NextPrevStyle ForeColor="Black" VerticalAlign="Bottom"></NextPrevStyle>
-    <DayHeaderStyle Font-Bold="True" BackColor="#F0F0F0"></DayHeaderStyle>
-    <SelectedDayStyle Font-Bold="True" Font-Underline="True" ForeColor="Black" BackColor="#C5E0FC"></SelectedDayStyle>
-    <TitleStyle ForeColor="Black" Font-Bold="True" BorderColor="Black" BackColor="#C5E0FC"></TitleStyle>
-    <WeekendDayStyle BackColor="#F0F0F0"></WeekendDayStyle>
-    <OtherMonthDayStyle ForeColor="Gray"></OtherMonthDayStyle>
+    <DayHeaderStyle Font-Bold="True" BackColor="#E1E1E1"></DayHeaderStyle>
```

Comment:
I think we should do the header or the current date in primary color. please suggest which one based on the themes and personal reasoning. I have a hunch about what is better, but I'm looking for up-front input. two screenshots welcome.

Reply:
# Primary
![datepicker_primary_header](https://user-images.githubusercontent.com/48289242/129735766-c8256def-6b81-4251-b4b8-493ed2c7824e.png)
![datepicker_primary_today](https://user-images.githubusercontent.com/48289242/129735769-04086d72-37f4-4450-906f-f3248206f66b.png)

# Primary highlight
![datepicker_primary_highlight_header](https://user-images.githubusercontent.com/48289242/129735944-b4e08b74-3a2e-4bff-a0d9-a8c68b3baba6.png)
![datepicker_primary_highlight_today](https://user-images.githubusercontent.com/48289242/129735950-fd7a0e6a-a680-4e9a-b864-d28dc49ba72a.png)

# Indicator normal
![datepicker_indicator_normal_header](https://user-images.githubusercontent.com/48289242/129736031-6da17243-f02c-49d4-b07b-11ab32c6677b.png)
![datepicker_indicator_normal_today](https://user-images.githubusercontent.com/48289242/129736035-ca89ff0e-4616-4908-b021-b83a5f468ce7.png)

Reply:
I think `primary` is too dark. The `indicator normal` blue is too blue, and the `primary-highlight` color combination also does not look good. I think we should stay with the current colors.

Reply:
you need to switch the font color to pimary-text (=white). plz retry :)

Reply:
and indicator normal is nixed, it's not meant for this. I just need primary, same we will use for the primary button.

Reply:
hmm... I'm leaning towards the other version, simply because the primary color is meant to indicate the primary function / enter. So to stick with the style intention, I should use the seond one. But I do admit, the first one has a bit more charme.

Hmm... actually, we should try to use the dark-gray for the month line and the weekend-gray for the weekday-line. this might be even more in line with the intention and there's a halfway-precencende in the Outlook taskbar calender.

Reply:
![image](https://user-images.githubusercontent.com/48289242/129846607-f7f32607-a990-45a7-b04e-041d99feeda9.png)
is the color of the selected day ok?

Reply:
Colors look good. Please invert semantics:
Selected Date = primary + underline
Today = light blue, no underline